build-config/arch/armv8a.make

```diff
@@ -41,17 +43,26 @@ ifeq ($(UEFI_ENABLE),yes)
   FIRMWARE_TYPE = uefi
 endif
 
-PLATFORM_IMAGE_COMPLETE = $(IMAGE_UPDATER_STAMP) $(RECOVERY_ISO_STAMP)
+ifeq ($(UEFI_ENABLE),yes)
+  PLATFORM_IMAGE_COMPLETE = $(IMAGE_UPDATER_STAMP) $(RECOVERY_ISO_STAMP)
+else
+  PLATFORM_IMAGE_COMPLETE = $(IMAGE_BIN_STAMP) $(IMAGE_UPDATER_STAMP)
+endif
 
 PXE_EFI64_ENABLE ?= no
 
 ifeq ($(PXE_EFI64_ENABLE),yes)
   PLATFORM_IMAGE_COMPLETE += $(PXE_EFI64_STAMP)
 endif
 
-UPDATER_IMAGE_PARTS = $(UPDATER_VMLINUZ) $(UPDATER_INITRD) $(UPDATER_ONIE_TOOLS) \
+ifeq ($(UEFI_ENABLE),yes)
+  UPDATER_IMAGE_PARTS = $(UPDATER_VMLINUZ) $(UPDATER_INITRD) $(UPDATER_ONIE_TOOLS) \
 			$(ROOTCONFDIR)/grub-arch/sysroot-lib-onie/onie-blkdev-common
-UPDATER_IMAGE_PARTS_COMPLETE = $(KERNEL_INSTALL_STAMP) $(UPDATER_INITRD) $(UPDATER_ONIE_TOOLS)
+  UPDATER_IMAGE_PARTS_COMPLETE = $(KERNEL_INSTALL_STAMP) $(UPDATER_INITRD) $(UPDATER_ONIE_TOOLS)
+else
+  UPDATER_IMAGE_PARTS = $(UPDATER_ITB) $(UPDATER_UBOOT)
+  UPDATER_IMAGE_PARTS_COMPLETE = $(UPDATER_ITB) $(UBOOT_INSTALL_STAMP)
+endif
 
```

Comment:
This commit looks good.

It is a little misleading, in that we are implying "UEFI not enabled" is the same thing as "U-BOOT is enabled".  Minor.  This is OK.